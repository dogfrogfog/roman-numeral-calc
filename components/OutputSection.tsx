import styles from '@/styles/OutputSection.module.css'

type OutputSectionType = {
  romanNumber: string;
}

const OutputSection = ({ romanNumber }: OutputSectionType) => {
  return (
    <div className={styles.section}>
      <h3 className={styles.title}>Roman output</h3>
      <p className={styles.outputValue}>{romanNumber}</p>
    </div>
  )
}

export default OutputSection
