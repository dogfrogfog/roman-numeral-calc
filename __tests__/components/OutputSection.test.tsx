import { render } from '@testing-library/react'

import OutputSection from '@/components/OutputSection'

describe('Component: OutputSection', () => {
  it('renders component correctly', () => {
    const { container } = render(<OutputSection romanNumber='XX' />)

    expect(container).toMatchSnapshot()
  })
})
