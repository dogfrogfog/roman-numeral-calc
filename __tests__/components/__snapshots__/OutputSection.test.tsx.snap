// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: OutputSection renders component correctly 1`] = `
<div>
  <div
    class="section"
  >
    <h3
      class="title"
    >
      Roman output
    </h3>
    <p
      class="outputValue"
    >
      XX
    </p>
  </div>
</div>
`;
